win32/vcxproj/topaz_connect.vcxproj

```diff
@@ -88,7 +88,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
you've accidentally included solution changes

Reply:
ah shoot... how would I remove that? or block it from going forward?

Reply:
I believe in this case you can push an amend commit that changes it back to the exact original value. These files can be edited manually, but you may need to use an external text editor since Visual Studio is reluctant to let you open/edit project files by hand.

If you're going to retarget the solution, I don't think there's a way to prevent changes to your project file from being potentially included in git changes - it can't be added to `.gitignore` because sometimes the solution _does_ need to change. You can check the file changes before you commit and "revert" any changes that Visual Studio made after a retarget. (Alternatively: update your toolset so you don't need to retarget 😉)